action.yaml

```diff
@@ -2,11 +2,16 @@ name: Upload Charm Docs
 description: Upload charm documentation to charmhub
 author: David Andersson
 inputs:
-  create_new_topic:
+  delete_topics:
     description: |
-      Whether to create a new topic and commit it back to the repository if the
-      docs key in metadata.yaml either does not exist or is empty. Defaults to
-      not creating a new topic in which case an error is returned.
+      Whether to delete topics from the documentation server if they are no
```

Comment:
What does "not longer needed" mean here?

Reply:
So this is when, for example, a file is deleted in the repository which will usually result in the topic being deleted on discourse, unless `delete_topics` is set to false in which case the topic is left on discourse. This is based on feedback from the design document where people wanted the ability to have to action not delete topics on discourse